app/models/talking_stick/participant.rb

```diff
@@ -1,5 +1,6 @@
 module TalkingStick
   class Participant < ActiveRecord::Base
+    self.table_name = 'talking_stick_participants'
```

Comment:
You can probably borrow some of the [isolate_namespace](http://apidock.com/rails/Rails/Engine/isolate_namespace/class) implementation to avoid this kind of boilerplate.